lib/camera.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:image_picker/image_picker.dart';
+
+String videoPath, imagePath;
+
+class Camera {
+  openCamera() async {
+    print("camera::");
+    var _cameraImagePath = await ImagePicker.pickImage(
+      source: ImageSource.camera,
+    );
+    if (_cameraImagePath != null) imagePath = _cameraImagePath.path;
+  }
+
+  pickImage() async {
+    print("gallery:");
+    var _galerryImagePath = await ImagePicker.pickImage(
```

Comment:
pay attention to spelling. This should be galleryImagePath. Also no need for _ since this is a function variable